extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DevServicesBuildTimeConfig.java

```diff
@@ -10,18 +10,17 @@
 public interface DevServicesBuildTimeConfig {
 
     /**
-     * If DevServices has been explicitly enabled or disabled. DevServices is generally enabled
-     * by default, unless there is an existing configuration present.
+     * If DevServices has been explicitly enabled or disabled.
+     * DevServices is generally enabled by default unless an existing configuration is present.
      *
-     * When DevServices is enabled Quarkus will attempt to automatically configure and start
-     * a database when running in Dev or Test mode.
+     * When DevServices is enabled, Quarkus will attempt to configure and start automatically a database when running in Dev or Test mode.
```

Comment:
```suggestion
When DevServices is enabled, Quarkus will attempt to automatically configure and start a database when running in Dev or Test mode.
```

Not sure why the word "automatically" should be moved here.

Reply:
Cause I wanted to say:
* When DevServices is enabled, Quarkus will attempt to configure and start  a database automatically when running in Dev or Test mode.